app/views/foods/_table_row.html.erb

```diff
@@ -0,0 +1,10 @@
+<tr>
+  <td><%= food.name %></td>
+  <td><%= food.measurement_unit %></td>
+  <td><%= food.price %></td>
+  <td class="del_btn">
+    <%= button_to new_food_path(current_user, food), method: :delete, class: "del_btn" do %>
```

Comment:
The issue that's preventing you from deleting food is the path you're specifying, change it to this one, it would solve the issue: `food_path`.

Reply:
Already do the change